website/content/docs/concepts/storage.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+page_title: Storage
+sidebar_title: Storage
+description: >-
+  Vault relies on external storage to save its information.
+---
+
+# Storage
+
+As described on our [Architecture](/docs/internals/architecture) page, Vault's
+storage backend is untrusted storage used purely to keep encrypted information.
+
+## Supported Storage Backends
+
+For enterprise customers, HashiCorp provides official support for Consul and
```

Comment:
Do we want to remove "For enterprise customers" here?

Reply:
I don't think so? I think you have to be paying us to have support?

Reply:
My thought was that if support encompasses fixing bugs reported in the open and reviewing community PRs for Consul and Raft storage, it is free for non-enterprise folks.